src/components/actionBar/sortFilterRadioGroups/SortGroup.tsx

```diff
@@ -2,12 +2,30 @@ import { Radio, RadioGroup } from '@mui/material';
 import { StyledFormControlLabel } from '../filterSort/FilterSort.styled';
 import { useTranslation } from 'react-i18next';
 import { useSearchParams } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { setSort } from '../../../app/features/busSearchSlice';
 
 export default function SortGroup() {
     const { t } = useTranslation('filterSort');
+    const dispatch = useDispatch();
 
     const [searchParams, setSearchParams] = useSearchParams();
+
+    // upon mount the query param is set to store?
+
     const sortHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+        if (event.target.value === 'PriceHighToLow') {
```

Comment:
can we use switch?

Reply:
Changed the if-else ladder and used enum